assets/arc-0062/pyproject.toml

```diff
@@ -7,15 +7,15 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-algokit-utils = "^2.3.0"
+algokit-utils = "^3.0.1"
 python-dotenv = "^1.0.0"
-algorand-python = "^2.1.0"
-algorand-python-testing = "^0.4.1"
+algorand-python = "^2.7.0"
+algorand-python-testing = "^0.5.0"
 
 [tool.poetry.group.dev.dependencies]
-algokit-client-generator = "^1.1.6"
+algokit-client-generator = "^2.0.1b2"
```

Comment:
I would like to avoid merging a beta version algokit-client-generator = "^2.0.1b2"